equinix/provider_test.go

```diff
@@ -124,26 +123,6 @@ func (t *testAccConfig) build() string {
 	return t.config
 }
 
-// nprintf returns a string with all the placeholders replaced by the values from the params map
-//
-// Deprecated: nprintf is shared between NE resource tests and has been
-// centralized ahead of those NE resources moving to separate packages.
-// Use github.com/equinix/terraform-provider-equinix/internal/nprintf.NPrintf instead
-func nprintf(format string, params map[string]interface{}) string {
```

Comment:
swapping out the deprecated functions for their `comparisons` copies could be done in a separate PR ahead of this work

Reply:
In anycase, this commit message should be changed from "refactor: move nprintf to separate package" to "refactor: replace deprecated functions with comparisons pkg equivalents"